tweet/main.py

```diff
@@ -179,11 +179,17 @@ def tweet(cmd, param, words):
         else:
             bot.reply("That doesn't look like a link to a valid Tweet.")
     else:
-        results = send("statuses/update.json", status=bot.arguments)
-        user = results.get("user")
-        screen_name = user.get("screen_name")
-        tweet_id = results.get("id")
-        bot.reply(":boom: just tweeted it! :point_right: https://twitter.com/{}/status/{}".format(screen_name, tweet_id))
+        # Replace @@haacked with @haacked. This gives us an escape hatch when a twitter username matches Slack.
+        tweet_text = re.sub(r'@@([a-zA-Z0-9_]+)', r'@\1', bot.arguments) 

```

Comment:
Twitter usernames can only be between 1-15 characters long so you can probably do `@@([a-zA-Z0-9_]{1,15})` and be extra precise about it.

Reply:
Good call! I also need to do a look behind to make sure there’s a white space or beginning of string preceding `@@`.